pallets/grants/src/lib.rs

```diff
@@ -237,34 +280,26 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
 		fn build(&self) {
-			let grants = self
-				.vesting
-				.iter()
-				.map(|(ref who, schedules)| {
-					(
-						who.clone(),
-						schedules
-							.iter()
-							.map(|&(start, period, period_count, per_period)| VestingSchedule {
-								start,
-								period,
-								period_count,
-								per_period,
-							})
-							.collect::<Vec<_>>(),
-					)
-				})
-				.collect::<Vec<_>>();
-
-			// Create the required coins at genesis and add to storage
-			grants.iter().for_each(|(ref who, schedules)| {
-				let total_grants = schedules.iter().fold(Zero::zero(), |acc: BalanceOf<T>, s| {
+			self.vesting.iter().for_each(|(ref who, schedules)| {
+				let vesting_schedule: BoundedVec<VestingScheduleOf<T>, T::MaxSchedule> = schedules
+					.iter()
+					.map(|&(start, period, period_count, per_period)| VestingSchedule {
+						start,
+						period,
+						period_count,
+						per_period,
```

Comment:
Here we can indeed add unit tests to cover the lines that codecov has highlighted.